lib/rollout.rb

```diff
@@ -159,6 +166,33 @@ def define_group(group, &block)
     @groups[group.to_sym] = block
   end
 
+  def define_id_collection(name, ids = [])
+    @storage.set("collection:#{name}", ids.join(","))
+    @storage.set(collection_key, (collections.keys | [name.to_sym]).join(","))
+  end
+
+  def add_collection_to_feature(collection, feature)
+    with_feature(feature) do |feature|
+      @storage.set(collection_feature_key(feature.name), (feature_collections(feature.name).keys | [ collection.to_sym]).join(","))
+    end
+  end
+
+  def collections
+    Hash[*retrieve_from_storage(collection_key).map{ |collect| [ collect.to_sym, @storage.get("collection:#{collect}") ] }.flatten]
```

Comment:
:+1: much better :smile: .

The `.to_h` was only implanted on 
> 2.1 :disappointed: 

I will change for this approach \o

Thanks @GonchuB